src/utilities/hooks/ModalPaymentContext/types.ts

```diff
@@ -11,12 +11,24 @@
 
 import { BrowsePageSeller } from '../../api/types';
 
+export type lucData = {
+  firstName: string;
+  middleInitial: string;
+  lastName: string;
+  fullName: string;
```

Comment:
where is `fullName` set? Only seeing references to the other parts of the name

Reply:
fullname is set in the lantern form when adding the billing address